network/p2p/topic_validator_test.go

```diff
@@ -340,39 +343,38 @@ func TestAuthorizedSenderValidator_InvalidMsg(t *testing.T) {
 
 	unittest.RequireReturnsBefore(t, wg.Wait, 5*time.Second, "could not receive message on time")
 
-	// expecting 1 warn calls for each rejected message from ejected node
+	// expecting 1 error log for each rejected message from ejected node
 	require.Equalf(t, uint64(1), hookCalls, "expected 1 warning to be logged")
 }
 
 // TestAuthorizedSenderValidator_Ejected tests that the authorized sender validator rejects messages from nodes that are ejected
 func TestAuthorizedSenderValidator_Ejected(t *testing.T) {
 	sporkId := unittest.IdentifierFixture()
-	identity1, privateKey1 := unittest.IdentityWithNetworkingKeyFixture(unittest.WithRole(flow.RoleConsensus))
-	sn1 := createNode(t, identity1.NodeID, privateKey1, sporkId, zerolog.Nop())
 
-	identity2, privateKey2 := unittest.IdentityWithNetworkingKeyFixture(unittest.WithRole(flow.RoleConsensus))
-	sn2 := createNode(t, identity2.NodeID, privateKey2, sporkId, zerolog.Nop())
+	nodeFixtureCtx, nodeFixtureCtxCancel := context.WithCancel(context.Background())
+	defer nodeFixtureCtxCancel()
 
-	identity3, privateKey3 := unittest.IdentityWithNetworkingKeyFixture(unittest.WithRole(flow.RoleAccess))
-	an1 := createNode(t, identity3.NodeID, privateKey3, sporkId, zerolog.Nop())
+	sn1, identity1 := nodeFixture(t, nodeFixtureCtx, sporkId, "consensus_1", withRole(flow.RoleConsensus))
+	sn2, identity2 := nodeFixture(t, nodeFixtureCtx, sporkId, "consensus_2", withRole(flow.RoleConsensus))
+	an1, identity3 := nodeFixture(t, nodeFixtureCtx, sporkId, "access_1", withRole(flow.RoleAccess))
 
-	channel := network.ConsensusCommittee
-	topic := network.TopicFromChannel(channel, sporkId)
+	channel := channels.ConsensusCommittee
+	topic := channels.TopicFromChannel(channel, sporkId)
 
-	ids := flow.IdentityList{identity1, identity2, identity3}
+	ids := flow.IdentityList{&identity1, &identity2, &identity3}
 	translator, err := p2p.NewFixedTableIdentityTranslator(ids)
 	require.NoError(t, err)
 
 	// setup hooked logger
 	var hookCalls uint64
 	hook := zerolog.HookFunc(func(e *zerolog.Event, level zerolog.Level, message string) {
-		if level == zerolog.WarnLevel {
+		if level == zerolog.ErrorLevel {
```

Comment:
- updated test to use check log message strings directly https://github.com/onflow/flow-go/pull/2648/commits/62d6a270a998c5907e728a0dca43c38375f402fd
- using sentinels returned from the authorization config directly instead of wrapping them makes the above cleaner https://github.com/onflow/flow-go/pull/2648/commits/7ad61328d7b8fd66ee848943592f491cbe9a38c0